src/components/CircularProgress.tsx

```diff
@@ -0,0 +1,41 @@
+import { Stack, CircularProgress, CircularProgressProps } from '@mui/material';
```

Comment:
```suggestion
import { Stack, CircularProgress as CircularProgressMui, CircularProgressProps } from '@mui/material';
```

Para luego llamar al componente CircularProgress y no Centered ya que el centered es una prop que puede cambiar. Es una sugerencia, yo lo hice asi con avatar